tests/llmcompressor/test_kv_cache.py

```diff
@@ -0,0 +1,310 @@
+import os
+import tempfile
+
+import pytest
+from compressed_tensors.compressors import ModelCompressor
+from compressed_tensors.quantization.cache import KVCacheScaleType
+from compressed_tensors.quantization.utils.helpers import iter_named_quantizable_modules
+from datasets import load_dataset
+from transformers import AutoTokenizer
+
+from llmcompressor.transformers import SparseAutoModelForCausalLM, oneshot
+
+MODEL_IDS = [
+    "TinyLlama/TinyLlama-1.1B-Chat-v1.0",
+    "microsoft/Phi-3-mini-4k-instruct",
+]
+
+
+@pytest.fixture(scope="session")
+def oneshot_fixture():
+    num_bits = 8
+    _type = "float"
+    strategy = "tensor"
+    dynamic = False
+    symmetric = True
+
+    recipe = f"""
```

Comment:
It might be easier to just create a Recipe object, rather than having to serialize it into a string and dict